iset.mm

```diff
@@ -99583,6 +99597,769 @@ seq K ( .+ , G , S ) ) $=
+      WIVOEFVDRSAWAVSWDSAVSUKVOEDVEVFVGVHWKVIAWAWEVPVRVSVJWDWJDVOVLUQVM $.
+  $}
+
+  ${
+    $d J x $.  $d K x $.  $d M x $.
+    iseqf1olemklt.n $e |- ( ph -> N e. ( ZZ>= ` M ) ) $.
+    iseqf1olemklt.k $e |- ( ph -> K e. ( M ... N ) ) $.
+    iseqf1olemklt.j $e |- ( ph
+      -> J : ( M ... N ) -1-1-onto-> ( M ... N ) ) $.
+    iseqf1olemklt.const $e |- ( ph
+      -> A. x e. ( M ..^ K ) ( J ` x ) = x ) $.
+    iseqf1olemklt.kj $e |- ( ph -> K =/= ( `' J ` K ) ) $.
+    $( Lemma for ~ iseqf1o .  (Contributed by Jim Kingdon, 21-Aug-2022.) $)
+    iseqf1olemklt $p |- ( ph -> K < ( `' J ` K ) ) $=
+      ( cfv clt wbr wceq co wf1o wcel adantr syl neneqd wa f1ocnvfv2 syl2anc cv
+      ccnv cfz cfzo fveq2 id eqeq12d wral cuz cz wf f1ocnv f1of ffvelrnd elfzuz
+      elfzelz simpr elfzo2 syl3anbrc rspcdva eqtr3d mtand w3o ztri3or ecase23d
+      ) ADDCUFZLZMNZDVKOZVKDMNZADVKKUAZAVNVMVOAVNUBZVKCLZDVKVPEFUGPZVRCQZDVRRZV
+      QDOAVSVNISAVTVNHSVRVRDCUCUDVPBUEZCLZWAOZVQVKOBEDUHPZVKWAVKOZWBVQWAVKWAVKC
+      UIWEUJUKAWCBWDULVNJSVPVKEUMLRZDUNRZVNVKWDRAWFVNAVKVRRZWFAVRVRDVJAVRVRVJQZ
+      VRVRVJUOAVSWIIVRVRCUPTVRVRVJUQTHURZVKEFUSTSAWGVNAVTWGHDEFUTTZSAVNVAVKEDVB
+      VCVDVEVFAWGVKUNRZVLVMVNVGWKAWHWLWJVKEFUTTDVKVHUDVI $.
+  $}
+
+  ${
+    iseqf1olemqcl.k $e |- ( ph -> K e. ( M ... N ) ) $.
+    iseqf1olemqcl.j $e |- ( ph
+      -> J : ( M ... N ) -1-1-onto-> ( M ... N ) ) $.
+    iseqf1olemqcl.a $e |- ( ph -> A e. ( M ... N ) ) $.
+    $( Lemma for ~ iseqf1o .  (Contributed by Jim Kingdon, 27-Aug-2022.) $)
+    iseqf1olemqcl $p |- ( ph -> if ( A e. ( K ... ( `' J ` K ) ) ,
+        if ( A = K , K , ( J ` ( A - 1 ) ) ) ,
+        ( J ` A ) ) e. ( M ... N ) ) $=
+      ( cfv co wcel wa ad2antrr syl cz cle wbr elfzelz zred ccnv cfz wceq c1 wn
+      cmin cif wf wf1o f1of w3a elfzel1 elfzel2 peano2zm 3jca elfzle1 clt simpr
+      eqcom sylnib wo ad2antlr zleloe syl2anc mpbid zltlem1 letrd lem1d elfzle2
+      wb ecased jca elfz2 sylanbrc ffvelrnd zdceq adantr ifcldadc f1ocnv fzdcel
+      wdc 3syl syl3anc ) ABDDCUAZJZUBKLZBDUCZDBUDUFKZCJZUGBCJZEFUBKZAWFMZWGDWIW
+      KADWKLZWFWGGNWLWGUEZMZWKWKWHCAWKWKCUHZWFWNAWKWKCUIZWPHWKWKCUJOZNWOEPLZFPL
+      ZWHPLZUKEWHQRZWHFQRZMWHWKLWOWSWTXAWOWMWSAWMWFWNGNZDEFULOZWOWMWTXDDEFUMOZW
+      OBPLZXAAXGWFWNABWKLZXGIBEFSOZNZBUNOZUOWOXBXCWOEDWHWOEXETWODWOWMDPLZXDDEFS
+      ZOZTWOWHXKTZWOWMEDQRXDDEFUPOWODBUQRZDWHQRZWOXPDBUCZWOWGXRWLWNURBDUSUTWODB
+      QRZXPXRVAZWFXSAWNBDWEUPVBWOXLXGXSXTVJXNXJDBVCVDVEVKWOXLXGXPXQVJXNXJDBVFVD
+      VEVGWOWHBFXOWOBXJTZWOFXFTWOBYAVHWOXHBFQRAXHWFWNINBEFVIOVGVLWHEFVMVNVOAWGW
+      AZWFAXGXLYBXIAWMXLGXMOZBDVPVDVQVRAWJWKLWFUEAWKWKBCWRIVOVQAXGXLWEPLZWFWAXI
+      YCAWEWKLYDAWKWKDWDAWQWKWKWDUIWKWKWDUHHWKWKCVSWKWKWDUJWBGVOWEEFSOBDWEVTWCV
+      R $.
+
+    ${
+      $d A u $.  $d J u $.  $d K u $.  $d M u $.  $d N u $.
+      iseqf1olemqval.q $e |- Q = ( u e. ( M ... N ) |->
+        if ( u e. ( K ... ( `' J ` K ) ) ,
+          if ( u = K , K , ( J ` ( u - 1 ) ) ) ,
+          ( J ` u ) ) ) $.
```

Comment:
fits on single line

Reply:
The newline here is trying to make it easier to see which expression is part of the inner `if` and which expression is part of the outer `if`. I'm willing to combine those lines if you think we should, but at least to me this feels a bit different to some of these other cases.

Reply:
I think that if a line break is needed because a formula doesn't fit on a line, then the line break should occur at a semantically meaningful place, but if no line break is needed, then there is no reason to add one.